docs/tests/integration/test_vulnerability_detector/test_general_settings/test_general_settings_min_full_scan_interval.md

```diff
@@ -0,0 +1,59 @@
+# Test general settings min full scan interval
+
+The tests will modify the value of `min_full_scan_interval` tag in `ossec.conf`, 
+set different intervals and check the result in `ossec.log`.
+
+## General info
+
+|Tier | Number of tests | Time spent| Test file |
+|:--:|:--:|:--:|:--:|
+| 0 | 4 | 6m27s | test_general_settings_min_full_scan_interval.py
+
+## Test logic
+
+For different time units:
+
+```
+['s','m','h','d']
+```
+
+and the time value:
+
+```
+[120]
+```
+
+The test will check in `ossec.log` if the following message appears:
+
+```
+<vulnerability_detector_prefix> A full scan will be run on agent...
+```
+
+## Checks
+
+- [x] Vulnerability Detector does not launch a full scan of the agent even if the CVEs database is 
+  updated until the time set in `min_full_scan_interval` has expired.
+
+## Observed behavior
+
+The expected behavior is carried out.
+
+## Execution result
+
+```
+========================================== test session starts ===========================================
+platform linux -- Python 3.6.8, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
+rootdir: /vagrant/wazuh-qa/tests/integration, configfile: pytest.ini
+plugins: metadata-1.11.0, html-2.0.1, testinfra-5.0.0
+collected 4 items                                                                                      
+
+test_vulnerability_detector/test_general_settings/test_general_settings_min_full_scan_interval.py . [ 25%]
+...                                                                                                 [100%]
+
+===================================== 4 passed in 387.10s (0:06:27) ======================================
+```
```

Comment:
```suggestion
```

Follow the standard set in the `remoted` tests. Don't add the execution result to the docs. They are expected to pass.

Reply:
Done in: https://github.com/wazuh/wazuh-qa/pull/1369/commits/7438aaa4e055a855a5bd2cbd1fff70d33a42c43f